Makefile

```diff
@@ -161,23 +161,25 @@ test:
 release-update:
 	if [ ! -e RELEASE ]; then echo 0 > RELEASE; fi
 
-version-update: release-update
+.PHONY: ipasetup
+ipasetup:
+	sed -e s/__VERSION__/$(IPA_VERSION)/ ipasetup.py.in \
+		> ipasetup.py
+	$(PYTHON) setup.py egg_info
+	for directory in ipaclient ipalib ipaplatform ipapython ipatests; do \
+	    pushd $${directory} ; \
+	    cp ../ipasetup.py . ; \
+	    $(PYTHON) setup.py egg_info ; \
```

Comment:
IMO it would be better to run `setup.py` with `PYTHONPATH` set properly, rather than copy `ipasetup.py` all the time.